README.md

```diff
@@ -593,6 +593,7 @@ Name | Website | Region
 [Remote Garage](/company-profiles/remote-garage.md) | http://www.remotegarage.club/ | India
 [RemoteBase](/company-profiles/remotebase.md) | https://remotebase.com | Worldwide
 [RemoteMore](/company-profiles/remotemore.md) | https://remotemore.com | WorldWide
+[Renaissance](/company-profiles/renaissance.md) | https://www.renaissance.com/ | United States, UK
```

Comment:
I don't see anything on the site about UK so let's remove that and keep it as `United States`